docs/Makefile

```diff
@@ -1,3 +1,19 @@
+
```

Comment:
Empty line. But also, this file was auto-generated and the header may not be appropriate.